charts/matrix-antiscam-bot/Chart.yaml

```diff
@@ -0,0 +1,3 @@
+description: Web3 Foundation Matrix Antiscam Bot
+name: matrix-antiscam-bot 
+version: v0.0.1
```

Comment:
This repo is for publishing the charts only, the charts published from here are used later on `w3f-matrix-server` and `community-matrix-server`. There's a small issue in the way our tooling publishes charts, if they belong to the same repo they must all have the same version. So, if we want to publish this chart from this repo the version must match the `matrix-server` and `riot-web` charts' version.

Reply:
Ok thanks, updated to v0.5.0